### DISCUSSION TITLE
Error:   error starting container cimg/python:3.8: Error response from daemon: --storage-opt is supported only for overlay over xfs with 'pquota' mount option

### DISCUSSION THREAD

**Discussion by jomakori** [kienstra/localci#165] [Q&A] [answered]
@kienstra When attempting to run LocalCI jobs, I keep consistently receiving this error: 

```
ec2-user@ip-172-31-6-253 ~/devops (eks)>     /home/ec2-user/.vscode-server/extensions/localci.local-ci-1.7.8/bin/circleci/Linux/x64/bin/circleci local execute --job 'check-updated-files' --config /tmp/local-ci/devops/process.yml -v /tmp/local-ci/devops/volume:/tmp/local-ci
Fetching latest build environment...
Docker image digest: sha256:c3be9fb94fdf377e6a8d508fc0b833bb18a99a36a2d7d89a8e39c7701838f701
====>> Spin up environment
Build-agent version  ()
System information:
 Server Version: 20.10.13
 Storage Driver: overlay2
  Backing Filesystem: xfs
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Kernel Version: 4.14.281-212.502.amzn2.x86_64
 Operating System: Amazon Linux 2
 OSType: linux
 Architecture: x86_64

Starting container cimg/python:3.8
Error:   error starting container cimg/python:3.8: Error response from daemon: --storage-opt is supported only for overlay over xfs with 'pquota' mount option

Warning: No authentication provided, using CircleCI credentials for pulls from Docker Hub.
  image is cached as cimg/python:3.8, but refreshing...
3.8: Pulling from cimg/python
Digest: sha256:db17fcbb1469f1f70bbbd76867c6d371d4bf9596cae84ece3b62de8fe9c20fb2
Status: Image is up to date for cimg/python:3.8
Error: 
Unexpected environment preparation error: Error response from daemon: --storage-opt is supported only for overlay over xfs with 'pquota' mount option

Step failed
Task failed
```

**Comment by kienstra** [comment]
@jomakori, thanks so much for opening this and the log of the job. I'll look at this!

**Comment by kienstra** [comment, marked as answer]
Hi @jomakori,
It sounds like you were using a Mac when this happened, right?

Maybe you saw [this](https://support.circleci.com/hc/en-us/articles/7060937560859-How-to-resolve-error-storage-opt-is-supported-only-for-overlay-over-xfs-with-pquota-mount-option-when-running-jobs-locally-with-the-cli), but it's for Linux machines.

**Reply by jomakori** [reply]
Thanks! It works, I was running it via a linux-based cloud instance

**Reply by kienstra** [reply]
Ah, good to hear! So that [support article](https://support.circleci.com/hc/en-us/articles/7060937560859-How-to-resolve-error-storage-opt-is-supported-only-for-overlay-over-xfs-with-pquota-mount-option-when-running-jobs-locally-with-the-cli) fixed it?

**Reply by jomakori** [reply]
Yes!

**Reply by kienstra** [reply]
That's great, thanks for working through that!

**Comment by kienstra** [comment]
TODO in Local CI: display a VS Code notice when this error happens, with a link to the [support article](https://support.circleci.com/hc/en-us/articles/7060937560859-How-to-resolve-error-storage-opt-is-supported-only-for-overlay-over-xfs-with-pquota-mount-option-when-running-jobs-locally-with-the-cli).
